minify.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: minify
+  version: 2.12.8
+  epoch: 0
+  description: "Go minifiers for web formats"
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/tdewolff/minify
+      tag: v${{package.version}}
+      expected-commit: 65bbd3b7bcef06d21fac1af6a0ef7ff70a49800a
+
+  - uses: go/build
+    with:
+      packages: ./cmd/minify
+      ldflags: -s -w
+      output: minify
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/share/bash-completion/completions/
```

Comment:
If we install the bash completions we usually try to do it in a separate sub package.

Reply:
Fixed